flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/decorators/JavaCmdJobManagerDecorator.java

```diff
@@ -61,7 +58,7 @@ public FlinkPod decorateFlinkPod(FlinkPod flinkPod) {
 
 		final Container mainContainerWithStartCmd = new ContainerBuilder(flinkPod.getMainContainer())
 			.withCommand(kubernetesJobManagerParameters.getContainerEntrypoint())
-			.withArgs(Arrays.asList(NATIVE_KUBERNETES_COMMAND, startCommand))
+			.withArgs(KubernetesUtils.getStartCommandWithBashWrapper(startCommand))
```

Comment:
It might be better to call `KubernetesUtils#getStartCommandWithBashWrapper` from inside `getJobManagerStartCommand`, and make `getJobManagerStartCommand` returns an array. This would ensure the start command is always wrapped with `bash -c`.